ajax/libs/es6-tween

```diff
@@ -0,0 +1,44 @@
+{
+	"name": "es6-tween",
+	"description": "ES6 implementation of amazing tween.js",
+	"filename": "dist/Tween.min.js",
+	"keywords": [
+		"tween",
+		"tweening",
+		"es6",
+		"numeric",
+		"interpolation",
+		"easing",
+		"emit"
+	],
+	"repository": {
+		"type": "git",
+		"url": "git://github.com/tweenjs/es6-tween.git"
+	},
+	"author": "tween.js contributors",
+	"license": "MIT",
+	"npmName": "es6-tween",
+	"npmFileMap": {
+		"basePath": "dist",
+		"files": [
+			"Tween.js",
+			"Tween.min.js",
+			"Tween.js.map",
+			"Tween.min.js.map"
+		]
+	},
+	"autoupdate": {
```

Comment:
Please delete [#L30-L43](https://github.com/dalisoft/cdnjs/blob/83bb9ae0f47ef1c16893aed841e43dd381fdc716/ajax/libs/es6-tween#L30-L43), because we just need one auto-update, and there are more versions in npm and it has the new version.